core/jazz_ui/e2e/page-objects/jazzservices.po.ts

```diff
@@ -76,7 +76,8 @@ export class Jazz {
      return element(by.css('section.footer-btn>btn-jazz-primary'));
     }
   getServiceNameHeader() {
-     return element(by.css('div.page-title-wrap.hide-for-mobile>h1.page-hdr.bold-title'));
+     
+	 return element(by.xpath('//div[@class="page-title-wrap hide-for-mobile"]/h1[@class="page-hdr bold-title"]'));
```

Comment:
I wonder why the browser was not finding these elements on load. Were you able to find more details on this? Just curious. 
But I agree with you that for now, we can go with xpath where finding by element id is not trivial.

Reply:
I was trying both Parth,but may be some browser specific issues,I didn't go in deep since I am already running out of time

Reply:
understood... no problems